README.md

```diff
@@ -40,21 +40,22 @@ just install [xDSL via pip](https://pypi.org/project/xdsl/):
 pip install xdsl
 ```
 
-## Using xDSL
+## Getting Started
 
-To use xDSL we recommend following the [xDSL Developer Setup](#xdsl-developer-setup)
-to clone the repository including the notebooks.
-The following tutorials will present xDSL basic concepts, how to
-use its irdl dialect to define new dialects in a user-friendly way, and how to
-work with both xDSL and MLIR.
+To get into xDSL, we recommend our jupyter notebooks as the starting point. The
+notebooks consist of a small example of a database compiler and documentation on
+the core xDSL data structures and on xDSL's Python embedded abstraction
+definition language. There also exists a small documentation showing how to
+connect xDSL with MLIR for users interested in that use case.
 
+- [A Database example](https://xdsl.dev/xdsl/retro/notebooks/?path=database_example.ipynb)
```

Comment:
For me, the database example is more of a motivational example rather than a technical one, that's why I prefer to have it first. It is also written in a way that briefly explains some of the core concepts (what is an Operation, what are SSAValues...), such that it is usable and understandable without having to know all the core concepts.

Reply:
I guess this discussion comes down to what is each notebook supposed to do. For me that's:
- database: give motivation to keep reading by implementing a small DSL in a very quick way
- tutorial: give an overview of the data structures in xdsl (operations, regions, blocks, ssavalues...)
- irdl: give an overview over the decorators and how to apply them to build your own abstractions.

Clearly, tutorial should be before irdl. the database one is not quite as clear IMO.

Reply:
I am of the opinion that any guide should go from general to specific. But I am happy to get other opinions

Reply:
I would also love the idea to number the tutorials so as to show order of learning/execution rather than having them in alphabetical order. Example: https://github.com/devitocodes/devito/tree/master/examples/compiler